src/blob.js

```diff
@@ -31,13 +31,13 @@ export default class Blob {
 				if (element instanceof Buffer) {
 					buffer = element;
 				} else if (ArrayBuffer.isView(element)) {
-					buffer = new Buffer(new Uint8Array(element.buffer, element.byteOffset, element.byteLength));
+					buffer = Buffer.from(new Uint8Array(element.buffer, element.byteOffset, element.byteLength));
```

Comment:
`Buffer.from(element.buffer, element.byteOffset, element.byteLength)` should be enough. This way you are not copying the TypedArray twice (here and in the `Buffer.concat` below).